### PR TITLE
Tweak transfer gc rule

### PR DESCRIPTION
**Pull request by platfowner**
Change sumary:
- Reduce min_gc_num_siblings_deleted to 10
- Tweak transfer gc rule

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1181

### CODE REVIEW

File: blockchain-configs/mainnet-prod/timer_flags.json [hunk 1]
```diff
@@ -46,5 +46,13 @@
   "update_min_gc_num_siblings_deleted": {
     "enabled_block": 1414800,
     "has_bandage": true
+  },
+  "update_min_gc_num_siblings_deleted2": {
+    "enabled_block": null,
```
Comment (woomurf) [inline blockchain-configs/mainnet-prod/timer_flags.json:51]: If enabled_block is null, how is it applied?

File: db/bandage-files/update_min_gc_num_siblings_deleted2.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (woomurf) [inline db/bandage-files/update_min_gc_num_siblings_deleted2.js:1]: minor) How about update file name like `decrease_to_10_min_gc_num_siblings_deleted`?

File: blockchain-configs/mainnet-prod/timer_flags.json (hunk 1 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/timer_flags.json:51]: > If enabled_block is null, how is it applied?

These null values will be replaced with some numbers when they're actually deployed in the release process.

Comment (woomurf) [reply blockchain-configs/mainnet-prod/timer_flags.json:51]: Thank you for explaining!

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

File: db/bandage-files/update_min_gc_num_siblings_deleted2.js (hunk 2 above)
Comment (platfowner) [reply db/bandage-files/update_min_gc_num_siblings_deleted2.js:1]: Thanks for the suggestion. For this, let's keep the other parts just simple highlighting the key pram updated.

**Comment by platfowner** [comment]
Thanks for the review. Now merging.. @woomurf

**Pull request merged**